frontend/hooks/useMultisig.ts

```diff
@@ -42,20 +43,21 @@ export const useMultisig = (safe?: Safe) => {
 /**
  * Hook to fetch from an array of multisigs
  */
-export const useMultisigs = (safes: Safe[]) => {
+export const useMultisigs = (safes?: Safe[]) => {
   const {
     data: owners,
     isFetched: ownersIsFetched,
     isPending: ownersIsPending,
   } = useQuery<{ safeAddress: string; chainId: number; owners: string[] }[]>({
-    queryKey: REACT_QUERY_KEYS.MULTISIGS_GET_OWNERS_KEY(safes),
+    queryKey: safes ? REACT_QUERY_KEYS.MULTISIGS_GET_OWNERS_KEY(safes) : [],
```

Comment:
there's enabled property, might be more suitable for this case? so that the queryFn won't be called until the saves are there https://tanstack.com/query/v4/docs/framework/react/guides/disabling-queries